rust/src/main.rs

```diff
@@ -208,10 +223,16 @@ fn main() {
     selenium_manager
         .set_timeout(cli.timeout)
         .and_then(|_| selenium_manager.set_proxy(cli.proxy.unwrap_or_default()))
+        .and_then(|_| selenium_manager.stats())
```

Comment:
Could we move the stats creation and upload at the end? The "primary" tasks of SM should go first and these type of things at the end, ideally to prevent any issues. 

I mean, if there is an issue with the `stats()` code, would the execution continue?

Reply:
The call to Plausible (`.stats()`) is done in a different thread. This way, even if Plausible is offline or any other unexpected error happens, the regular behavior of Selenium Manager will not be affected since the `.setup()` logic is executed in the main thread. I have tested it.

Reply:
👍